addons/ui/RscDisplayOptionsLayout.hpp

```diff
@@ -12,7 +12,7 @@ class RscDisplayOptionsLayout {
             onMouseButtonUp = "with uiNamespace do {['mouseButtonUp',_this,''] call RscDisplayOptionsLayout_script;};";
         };
         #define ADD_ELEMENT(var1) class Element##var1: Element022 {\
-            idc = 12##var1;\
+            idc = QUOTE(12##var1);\
```

Comment:
throws error on this
not sure if technically valid or not, it's very clever but strange
but it's no problem at all to just throw in quotes

Reply:
HEMTT leaves the joins in for the config parser to see, it could be fixed there. With a `var1` of `01`, the config parser wouldn't see `1201`, it would see `12##01`, and it only expects `##` in the context of identifiers, not numbers.

Reply:
I think this is getting fixed in newest hemmt, will test

Reply:
Correct, shouldn't need a `QUOTE` either.

Reply:
Confirmed fixed.